Marlin/src/Marlin.cpp

```diff
@@ -217,7 +217,7 @@ void setup_killpin() {
 
 void setup_powerhold() {
   #if HAS_SUICIDE
-    OUT_WRITE(SUICIDE_PIN, HIGH);
+    OUT_WRITE(SUICIDE_PIN, !SUICIDE_PIN_INVERTING);
```

Comment:
Why not call `suicide()` instead?

Reply:
I didnt go over all usage points, probably needs some review why M81 and suicide are opposite since that doesnt seem right.

Reply:
Ops I marked wrong code.... `suicide` write LOW, here you write HIGH..sorry